src/python/foglamp/update_statistics_history.py

```diff
@@ -0,0 +1,139 @@
+"""
```

Comment:
The first line of a python file should not be the module docstring. Please see \_\_template\_\_.py https://foglamp.readthedocs.io/en/develop/dev-tools.html#python-template

It should start with: 

    # -*- coding: utf-8 -*-
    
    # FOGLAMP_BEGIN
    # See: http://foglamp.readthedocs.io/
    # FOGLAMP_END